client/src/components/CreateReport/FeedBackModal.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Modal } from "antd";
+import { useDispatch } from "react-redux";
+import { useTranslation } from "react-i18next";
+
+import { postsActions } from "reducers/posts";
+import { theme } from "constants/theme";
+import { Footer, Submit } from "components/CreateReport/Body";
+const { colors, typography } = theme;
+
+const FeedBackModal = ({
```

Comment:
this modal name is a little confused with the `Feedback` in the user Profile dropdown

Reply:
will rename